less/datepicker.less

```diff
@@ -5,7 +5,7 @@
 		width: 220px;
 	}
 	direction: ltr;
-	&&-rtl {
+	&-rtl {
```

Comment:
Don't think much about it. Tho I'm not really sure to know how it can BC.   
jQuery/JS class selectors will still work since the classes will still be here.  
If someone has a CSS rule override ours, it will still works since this new rules has less weight (010/020/010) than the one before (020/030/020).